src/WebJobs.Extensions.DurableTask/HttpApiHandler.cs

```diff
@@ -46,6 +46,8 @@ internal class HttpApiHandler : IDisposable
         private const string ShowHistoryParameter = "showHistory";
         private const string ShowHistoryOutputParameter = "showHistoryOutput";
         private const string ShowInputParameter = "showInput";
+        private const string ShowHistoryInputParameter = "showHistoryInput";
+        private const string InstanceFilterOption = "instanceFilterOption";

```

Comment:
With my suggestion to reuse `showInput`, I think we can revert all changes to this file.